### ISSUE TITLE
Add Desktop in alt+tab options

### ISSUE THREAD

**Issue by drdhaval2785** [JeffHoogland/moksha#98]
Currently only open applications are shown in alt+tab.
In other distros / Desktop environments, the alt+tab also shows up desktop.
If I have some file / application icon placed on desktop, I would not be able to reach to it without moving mouse, once I have some application opened which covers the desktop.
Attaching the screencast.

[desktop_not_included_in_alt_tab.mp4.gz](https://github.com/JeffHoogland/moksha/files/3141444/desktop_not_included_in_alt_tab.mp4.gz)

**Comment by thewaiter** [comment]
This should be very easy to solve. Open Windows switcher and check the first option "Windows from other desk". See the picture
https://i.imgur.com/c19KHQg.jpg

**Comment by thewaiter** [comment]
Anyway, maybe you could use Linuxquestion forum for these kinds of "issues". We will kindly answer there it it will create info base for other users. 

Stefan

**Comment by drdhaval2785** [comment]
I thought that it was something specific to Moksha desktop environment, so asked here. Next time will think slightly more before posting here.

**Comment by rbtylee** [comment]
It is ok to post suspected issues here. As to your issue, it is not entirely clear to me what you are asking for. My first thought upon reading it was that you want an option to Show the Desktop added to the Window Switcher Module? In other words add some of the functionality found in the Module, [Deskshow](https://github.com/JeffHoogland/moksha-modules-extra/tree/master/deskshow).

Are you aware of this module? It can installed in Bodhi Linux 5.0 by

`sudo apt install moksha-module-deskshow`

Once installed it can added as a gadget to a shelf or the desktop and works by clicking it. But if you prefer one can create a key binding for it.

Is this what you are wishing was present in the window selection (ALT + TAB) dialog?

**Comment by thewaiter** [comment]
Hmm, did I understand your first post? Now I am confused. If you just need to show the desktop just press CTRL ALT D as default shortcut. Tell us more about your meaning please...

**Comment by drdhaval2785** [comment]
Let me try to rephrase it to make it clear.

How to reproduce:
1. In windows (and probably in ubuntu), open any application let's say text editor.
2. Click on alt+tab.
3. You will see two items in window switcher (1) text editor, (2) desktop. 
4. Do steps 1 and 2 in Bodhi / Moksha.
5. You will see only one item i.e. text editor. 

So if your desktop icons are placed behind the text editor, you need to move it via cursor or CTRL ALT D as suggested above.

Windows etc have desktop added by default in window switcher module. Can we have this in Bodhi please?

**Comment by thewaiter** [comment]
It is clear now for me. Need to do some research but I am not sure this is possible in current Moksha desktop.

**Comment by thewaiter** [comment]
OK, I played with the winlist code. I added "d" key to show desktop in case winlist is opened. Is this enough for you or you need the winlist item for showing the desktop? More difficult of course for me.

**Comment by drdhaval2785** [comment]
I am not very familiar with the code of Moksha. So not sure whether your solution would work or not till I see it in action. If I get Desktop option as in windows / ubuntu, my issue is resolved. How it is achieved / achievable is not material for me as an end user. It is the programmer's choice. I have no issues.

**Comment by thewaiter** [comment]
The thing I can do is making a moksha deb package for testing and send you via mail or upload to a cloud.

**Comment by drdhaval2785** [comment]
Sounds good to me

**Comment by drdhaval2785** [comment]
Will give it a try.

**Comment by thewaiter** [comment]
Are you using Bodhi Linux 5? If yes, send me a mail to thewaiter@centrum.sk. I will reply with moksha.deb file

**Comment by thewaiter** [comment]
Have you received my mail? I sent 2 days ago...
